packages/widget/src/vite-env.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="vite/client" />
+
+interface ImportMetaEnv {
+  readonly VITE_BRIDGE_ENV: string;
```

Comment:
why not `Environment` instead of `string`? this way those type assertions in the next file become redundant + we have more type safety

Reply:
Can you guarantee env will always be enum value?
 I'm usually quite against external API relying on typescript types as you might have a javascript user that can submit anything and you should validate external inputs (even from developer)

Reply:
i'd definitely leave the validation in `connectedCallback`, but yeah, makes sense to leave input as `string` since it can be anything. but then, we can immediately (when setting `@property environment`) check if it is an enum value and fallback to mainnet + throw (in `connectedCallback`), so prop is always of type `Environment`? 🤷